src/index.ts

```diff
@@ -112,7 +112,7 @@ const userAgentRules: UserAgentRule[] = [
   ['safari', /Version\/([0-9\._]+).*Safari/],
   ['facebook', /FBAV\/([0-9\.]+)/],
   ['instagram', /Instagram\s([0-9\.]+)/],
-  ['ios-webview', /AppleWebKit\/([0-9\.]+).*Mobile/],
+  ['ios-webview', /AppleWebKit\/([0-9\.]+).*Gecko\)/],
```

Comment:
Just remove `Mobile` at the end of the regex here (i.e. don't add `Gecko` in as a replacement).

Reply:
Actually, I think that's bad advice (sorry).  Just been looking at https://deviceatlas.com/blog/mobile-browser-user-agent-strings (for want of trying to find a better source) and I can see that we are probably going to get false positives on other browsers :/

User agent parsing... who'd do it?

Reply:
It's likely the `Gecko` change isn't going to fly either as this is the UA returned from my desktop chrome instance I'm using now:

```
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.119 Safari/537.36
```

Reply:
Ok, so it seems as though the new iOS Webview user agent is the only one to end on `Gecko)`. So I could instead make a new regex leaving the original one is it is:
```
['ios-webview', /AppleWebKit\/([0-9\.]+).*Mobile/],
['ios-webview', /AppleWebKit\/([0-9\.]+).*Gecko\)$/],
```